promotions/lib/generators/solidus_promotions/install/templates/initializer.rb

```diff
@@ -0,0 +1,12 @@
+# frozen_string_literal: true
+
+# Make sure we use Spree::SimpleOrderContents
+# Spree::Config.order_contents_class = "Spree::SimpleOrderContents"
+# Set the promotion configuration to ours
+# Spree::Config.promotions = SolidusPromotions.configuration
+
+# SolidusPromotions.configure do |config|
+#   Sync order promotions between the legacy and new promotion systems
```

Comment:
We can link to the configuration file instead.

Reply:
Done